### ISSUE TITLE
Unsupport FP16 ONNX | 不支持 FP16 ONNX 推理

### ISSUE THREAD

**Issue by laugh12321** [CVHub520/X-AnyLabeling#250]
Taking YOLOv5 as an example, in actual testing, only the model exporting FP32 can reason normally, while FP16 inference is abnormal

**Comment by CVHub520** [comment]
Yes, you're correct. I apologize for any inconvenience, but currently, we don't support FP16 inference yet.

**Comment by laugh12321** [comment]
Thanks
